shlr/java/dsojson.c

```diff
@@ -580,6 +575,7 @@ R_API int dso_json_dict_insert_key_obj (DsoJsonObj *dict, DsoJsonObj *key, DsoJs
 		r_list_append (the_list, entry);
 		res = true;
 	} else {
+		dso_json_obj_del (value);
```

Comment:
@radare no it is not leaking because I called that same function right before freeing the pointer so I assume no leaking happens